lib/tasks_private/yarn.rake

```diff
@@ -0,0 +1,7 @@
+namespace :yarn do
```

Comment:
Is this still needed?  If so, should this be a common task pushed down to the provider generator?

Reply:
The main ui-classic `update:yarn` already does `yarn install` for all plugins that bring js packages so I think we can remove this

Reply:
Ok, I'll have a follow up for these.